README.md

```diff
@@ -1,78 +1,14 @@
-# nextstrain.org/dengue
+# Nextstrain repository for dengue virus
 
-This is the [Nextstrain](https://nextstrain.org) build for dengue, visible at
-[nextstrain.org/dengue](https://nextstrain.org/dengue).
+[![pre-commit.ci status](https://results.pre-commit.ci/badge/github/nextstrain/dengue/main.svg)](https://results.pre-commit.ci/latest/github/nextstrain/dengue/main)
```

Comment:
Since we are not using pre-commit CI in this repo, I recommend removing this to prevent any confusion.

Reply:
Dropped with commit https://github.com/nextstrain/dengue/pull/15/commits/71a332c71f96d1610b54fc119385f14b8b24aa40